sources/packages/web/src/components/common/notes/Notes.vue

```diff
@@ -104,7 +104,7 @@ export default defineComponent({
     },
   },
   emits: ["submitData"],
-  setup(props: any, context: any) {
+  setup(_props, context: any) {
```

Comment:
Context is typed to any, please remove the explicit typing.

https://github.com/bcgov/SIMS/pull/1916/files#r1184391057